addons/survey/wizard/survey_invite.py

```diff
@@ -25,7 +25,7 @@ def _get_default_author(self):
     # composer content
     attachment_ids = fields.Many2many(
         'ir.attachment', 'survey_mail_compose_message_ir_attachments_rel', 'wizard_id', 'attachment_id',
-        string='Attachments')
+        string='Attachments', related='template_id.attachment_ids', store=True)
```

Comment:
this should be `readonly=True` so the attachments are still editable (and we use `template_id.attachment_ids` only as a default`)